m4_sessions/1801-inning/Group_1.md

```diff
@@ -1,3 +1,39 @@
+#Update Group 1: Ethics in AI
+
+## Why should students pick this session?
+To equip themselves to understand the ethics and problems revolving around artificial intelligence - one of the most important issues of the next century.
+
+## Discuss
+* What is AI?
+* For the people who are worried about it, what are they worried about?
+
+## Issues
+#### Human dignity
+The effect on us of losing human interactions. Joseph Wasenbaum - "AI technology should not be used to replace positions that require respect and care. Customer service reps, nursemaids for the elderly, police officers. If machines replace them, we will find ourself alienated...." Counter argument from Pamela McCordic.
+#### Implications on security/privacy
+“I like my Alexa, but I don’t like being recorded and monitored.”
+#### Automation > Labor
+How do we manage this as a society? Is it worth it? Charles Murray.
+#### Difficulty verifying/pinpointing mistakes
+Black Box Problem (https://www.technologyreview.com/s/604087/the-dark-secret-at-the-heart-of-ai/)
+  
+#### Other Topics
+* Machine consciousness
+* How does this relate to us as developers of software? (Developers that write code that writes code...). 
+* Human emotion is flawed, is this why we need machines?
+https://www.forbes.com/sites/tonybradley/2017/07/31/facebook-ai-creates-its-own-language-in-creepy-preview-of-our-potential-future/#131c0c3e292c
+
+
+## Timeline
```

Comment:
Would love to know which one of you is doing each part and what your main talking points are for each one.  This is going to be a GREAT session, but be sure you are prepared for all of these discussions.